internal/sdk.go

```diff
@@ -43,6 +43,7 @@ type Sdk struct {
 	Plugin     *LuaPlugin
 	// current sdk install path
 	InstallPath string
+	Name        string
```

Comment:
You don't need to add this field, and can get `sdk-name` from Plugin#Filename.

Reply:
ok, it actually is useless now, I add this property because I move the `select ui` into the Sdk, the ui should show a name about current sdk